cli/azd/pkg/input/inputhelper/helper.go

```diff
@@ -0,0 +1,115 @@
+package inputhelper
```

Comment:
I am wondering if this new package is necessary, see my other comments in this file.

Reply:
Needed to move these into a different module.  It was causing some circular references with other azure related modules.  Makes sense to move this into `azureutil` or similar.